.travis.yml

```diff
@@ -9,10 +9,9 @@ before_install:
 script: make
 
 rvm:
-  - 2.2.2
-  - 2.3.3
-  - 2.4.1
-  - 2.5.0
+  - 2.3.8
```

Comment:
I updated the rubies to recent point releases, and dropped 2.2.x since it's now EOL. The matrix is already huge so we can't realistically test EOL versions.